style/properties/shorthands/text.mako.rs

```diff
@@ -8,6 +8,7 @@
                     engines="gecko servo"
                     flags="SHORTHAND_IN_GETCS"
                     sub_properties="text-decoration-line
+                    ${' text-decoration-style text-decoration-color' if engine == 'servo' else ''}
```

Comment:
Just take common longhands (style and color) outside of the conditional, like `text-decoration-line`.

Reply:
done